### ISSUE TITLE
`querystring.parse()` change in v0.10

### ISSUE THREAD

**Issue by OrangeDog** [joyent/node#14001]
Probably old news, but this wasn't mentioned at [Api changes between v0.8 and v0.10](https://github.com/joyent/node/wiki/Api-changes-between-v0.8-and-v0.10).
Was this deliberate? What exactly has changed? How can I get the old behaviour, assuming doing so is at all sensible?

    > process.version
    'v0.8.22'
    > require('querystring').parse(Buffer([0xC3, 0x84, 0xC2, 0x90]).toString('ascii'))
    { 'Ä': '' } 

    > process.version
    'v0.10.37'
    > require('querystring').parse(Buffer([0xC3, 0x84, 0xC2, 0x90]).toString('ascii'))
    { 'C\u0004B\u0010': '' }

**Comment by KenanY** [comment]
Result of `git bisect`:

```
commit 96a314b68b19277cc71ccac06b6517956b8f8a22
Author: Ben Noordhuis <info@bnoordhuis.nl>
Date:   Thu Dec 6 05:13:14 2012 +0100

    buffer: strip high bits when converting to ascii
    
    Consider the following example:
    
      console.log(Buffer('ú').toString('ascii'));
    
    Before this commit, the contents of the buffer was used as-is and hence it
    prints 'ú'.
    
    Now, it prints 'C:'. Perhaps not much of an improvement but it conforms to what
    the documentation says it does: strip off the high bits.
    
    Fixes #4371.

:040000 040000 c79757ea5c9354200ad690734692a7520888f853 307abc26c78057576d0f34c53f189f4203ab2f32 M	doc
:040000 040000 d7fd469209e5129142a0a1a5cfd997a016e1bfd6 c38f913d6ed787c2160cb25f500f8a9e78d70f41 M	src
:040000 040000 660c440b099351d592e2c620092899148f013547 9ec9b28420e0f86a6cae59f2e8fa8fc714dc551d M	test
```

**Comment by OrangeDog** [comment]
Thanks. Don't know why I didn't think to check the `toString()` result.
Therefore, using `'binary'` instead of `'ascii'` should give the original behaviour.
